File: backend/ConstrutoraDesbravador.API/src/ConstrutoraDesbravador.Business/Models/Projeto.cs
using ConstrutoraDesbravador.Business.Enums;

namespace ConstrutoraDesbravador.Business.Models
{
    public class Projeto: Entity
    {
        public string Nome { get; set; }
        public string Descricao { get; set; }
        public DateTime DataInicio { get; set; }
        public DateTime DataTermino { get; set; }
        public StatusProjetoEnum StatusProjeto { get; set; }
        public RiscoProjetoEnum RiscoProjeto { get; set; }
        public int ResponsavelId { get; set; }
        public Funcionario Responsavel { get; set; }

        public ICollection<Funcionario> Funcionarios { get; set; }
    }
}

File: frontend/construtora-desbravador-spa/src/interfaces/Projeto.ts
import { RiscoProjetoEnum } from "../enums/RiscoProjetoEnum";
import { StatusProjetoEnum } from "../enums/StatusProjetoEnum";
import { Funcionario } from "./Funcionario";

export interface Projeto {
    id: number;
    nome: string;
    descricao: string;
    dataInicio: Date;
    dataTermino: Date;
    statusProjeto: StatusProjetoEnum;
    riscoProjeto: RiscoProjetoEnum;
    responsavel: Funcionario;
    funcionarios: Funcionario[]
}